src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -91,16 +103,44 @@ class DialogFullScreen extends Modal {
   }
 
   /**
-   * Returns title prop wrapped in <h2> if title is a string otherwise returns the title prop directly.
-   * Dialog main classes.
+   * Overrides the original function to disable the document's scroll.
+   */
+  get onOpening() {
+    Browser.addClass(this.document.body, DIALOG_OPEN_BODY_CLASS);
```

Comment:
Do we need a `Browser.addClass` function? We could just use `this.document.body.classList.add(DIALOG_OPEN_BODY_CLASS)`.

And is setting a class the right way to go here? e.g. as the `<body>` isn't actually the dialog element. Would using a data attribute be better? e.g. `data-fullscreendialogopen`.